src/computability/language.lean

```diff
@@ -151,4 +151,98 @@ begin
     finish }
 end
 
+lemma le_iff (l m : language α) : l ≤ m ↔ l + m = m :=
+begin
+  rw [set.le_eq_subset, add_def],
+  split,
+  { intro h,
+    ext x,
+    simp only [set.mem_union_eq, or_iff_right_iff_imp],
+    tauto },
+  { intro h,
+    rw ←h,
+    exact set.subset_union_left l m }
+end
+
+lemma one_add_lang_lang_star_le_lang_star (l : language α) : 1 + l*l.star ≤ l.star :=
```

Comment:
You're already in the `language` namespace, I don't think you need `lang` in the name
```suggestion
lemma one_add_self_mul_star_le_star (l : language α) : 1 + l*l.star ≤ l.star :=
```